Habitica/src/aura/TaskHabitItem/TaskHabitItem.cmp

```diff
@@ -0,0 +1,42 @@
+<aura:component >    
+    <aura:attribute name="item" type="Task__c"/>
+	    
+    <aura:registerEvent name="showTaskUpdateWindow" type="c:UpdateTaskEvent"/>  
+    
+    <article class="slds-card slds-card_boundary">
+        <div class="slds-card__header slds-grid">
+            <header class="task-header-text slds-media slds-media_center slds-has-flexi-truncate">
+                <a href="#" class="slds-text-heading_small">{!v.item.Name}</a>
```

Comment:
is there a need to have the href attribute here? since it leads to "#" it might just mean the wrong thing.